Explain --dump-config gives access via diagnostic server (#7043)
Kong/docs.konghq.com: app/_src/kubernetes-ingress-controller/reference/troubleshooting.md

Under normal operation, the controller does not store generated configuration;
it is only sent to Kong's Admin API. The `--dump-config` flag enables a
diagnostic mode where the controller also saves generated configuration to a

> temporary file and can be retrieved via web interface of the diagnostic
> server at `host:10256/debug/config`.

Reviewer: ```suggestion
temporary file. You can retrieve this file via the web interface of the diagnostic
server at `host:10256/debug/config`.
```

Author: Thanks
